Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
```

Comment:
There is not supposed to be a Pipfile at the root directory of the project, this repo houses several applications and each one has its own Pipfile (see [API](https://github.com/WordPress/openverse-api/blob/main/api/Pipfile) and [ingestion server](https://github.com/WordPress/openverse-api/blob/main/ingestion_server/Pipfile)). Please remove this and the Pipfile.lock that has also been committed. If you're having trouble getting `pre-commit` to run, check out the commands that our `justfile` uses:

https://github.com/WordPress/openverse-api/blob/a16947942ceaf3f1fbf81e5fc1e5d5275ce7976d/justfile#L109

Reply:
Alright, will remove the Pipfile.

Reply:
What will be the process of removing the files that have already been committed?